Character.cpp

```diff
@@ -50,11 +50,11 @@ const unsigned int Character::getDMG() const
 	return DMG;
 }
 
-void Character::gotHit(const Character* attacker)
+void Character::gotHit(const &Character attacker)

```

Comment:
Ti mit ittatok? Adjatok belole nekem is. :-D

Reply:
@hegyhati https://imgur.com/a/EtlrAWV

Reply:
Isten, Isten. Egeszseg.